Doc/library/csv.rst

```diff
@@ -59,7 +59,7 @@ The :mod:`csv` module defines the following functions:
    *csvfile* can be any object which supports the :term:`iterator` protocol and returns a
    string each time its :meth:`!__next__` method is called --- :term:`file objects
    <file object>` and list objects are both suitable.   If *csvfile* is a file object,
-   it should be opened with ``newline=''``. [1]_  An optional
+   it should be opened with ``newline=None``. [1]_  An optional
```

Comment:
``newline=''`` and ``newline=None`` both works fine for **csv.reader**. I think that this sentence just should be removed.